classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -612,7 +695,7 @@ protected function display_filter_by_status() {
 
 		if ( $status_list_items ) {
 			echo '<ul class="subsubsub">';
-			echo implode( " | \n", $status_list_items );
+			echo implode( " | \n", esc_html( $status_list_items ) );
```

Comment:
This triggers an error, breaking the Tools → Scheduled Actions screen entirely (because `implode` expects the second param to be an array—and passing the array through `esc_html()` gives us a string).

Each element is already [composed in a safe manner](https://github.com/woocommerce/action-scheduler/pull/763/files#diff-18ffca2c4125958b98cb71171cb6ddfc0ec7829f127c0094e24c6c2796cce326R693) and contains HTML, so `esc_html()` wouldn't be a good fit (since it would expose the HTML). 

Alternative:

```suggestion
			echo implode( " | \n", $status_list_items ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```